pages/api/google/export/roster.js

```diff
@@ -6,10 +6,10 @@ module.exports = async (req, res) => {
     if (req.method === 'POST') {
         
         // Get the Access Token from the request headers
-        const token = cookie.parse(req.headers.cookie).token;
-        const authStatus = await data.auth.checkAnyUser(`Bearer ${token}`, res);
```

Comment:
do we want to remove auth here

Reply:
No we don't, I will open a Part 3 Merge Request soon to add this back in.